src/IndProp.lidr

```diff
@@ -1,13 +1,13 @@
-= IndProp : Inductively Defined Propositions
+= IndProp: Inductively Defined Propositions
 
 > module IndProp
-
+>
```

Comment:
No, but I like it for completely silly reasons. I've using a cheap `awk` hack to generate `.idr` files, which helps me to visualize the module as a whole, without all the text.

```bash
$ cat >>Makefile <<EOF
%.idr: %.lidr
	@ awk '/^>/{ print substr($$0,3) }' $< >$*.idr
EOF
$ make -C src IndProp.idr
```